Code/GraphMol/MolDraw2D/Wrap/testMolDraw2D.py

```diff
@@ -839,6 +839,11 @@ def testGithub7409(self):
         # it's enough to check that this doesn't throw an exception
         d2d.DrawMolecule(m)
 
+    def testProtectedAttributes(self):
+        do = rdMolDraw2D.MolDrawOptions()
+        with self.assertRaises(AttributeError):
+            do.rhubarb = True
```

Comment:
what do you have against rhubarb? ;-)

Reply:
As Google summarises: "In British English, "he's talking rhubarb" means someone is speaking in a way that is nonsensical or doesn't make sense. It's often used in the context of crowd scenes in film or theater, where actors mumble "rhubarb, rhubarb" to create the sound of background chatter without making the specific dialogue audible"

I have only admiration for rhubarb the vegetable.